spec/grammar.dd

```diff
@@ -688,20 +693,24 @@ $(GNAME ForeachTypeAttributes)
 $(GNAME ForeachTypeAttribute):
     $(D ref)
     $(GLINK2 declaration, TypeCtor)
+    $(D enum)
```

Comment:
I wonder if `enum` should not be a `ForeachTypeAttribute`, but instead listed as a separate `ForeachType` entry, similar to `alias`.

Reply:
`enum` is a storage class, so it would be inconsistent not to allow it.